datafangst.md

```diff
@@ -2,6 +2,16 @@
 
 ## Leveranser 2020
 
+## Mars
+
+### Datafangst 2020-1.3.0
+* Datafangst støtter både NN54 og NN2000 som høydesystem inn til NVDB, og er klar for NVDBs overgang til NN2000.
+* Stedfesting bruker nå maks 8 desimaler på posisjon, slik at det blir mindre konflikt med stedfesting som allerede er i NVDB.
+* Automatisk fjerning av for tette punkter er skrudd på for innsending til NVDB, også for gamle data som ikke har fått dette gjort i Datafangst.
```

Comment:
Kanskje litt vanskelig å forstå hva dette betyr. Kanskje ", også for gamle data som ikke fikk dette gjort ved første import til datafangst." eller noe i den gaten?